homework/02LessonTwo.js

```diff
@@ -59,7 +59,8 @@ function add(number1, number2) {
 // this function will add three numbers
 // you must call your function add() in addThree()
  function addThree (number1, number2, number3) {
-   return number1 + number2 + number3;
+   var x= add(number1, number2)
+   return add(x, number3) 
```

Comment:
You also write:

"return number1 + number2 + number3" instead of using an extra variable, "x", to store your logic.